portfolio/src/main/webapp/leave-comment/leave-comment.html

```diff
@@ -12,18 +12,19 @@
   <body>
     <div id="leave-comment-form">
       Please share your feedback!
-      <form>
-        <input type="email" placeholder="Your email adress">
+      <form method="post" action="/add-comment">
```

Comment:
nit: I see that you're going for absolute paths after all. That's fine, but my pedantism would demand consistency in this area.

Reply:
Done.